### PR TITLE
i18n(pt-BR): Translate tutorial `setup-1` second half

### PR DESCRIPTION
**Pull request by Yan-Thomas**
#### What kind of changes does this PR include?

- Translated content

#### Description

PT-BR translation for the second half of `setup-1`.

### CODE REVIEW

File: src/content/docs/pt-br/tutorial/1-setup/4.mdx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+2. Clique no menu <kbd>•••</kbd> "3 pontos" acima da mensagem de commit e escolha <kbd>Remoto</kbd> > <kbd>Adicionar Remoto</kbd>.
+
+3. Selecione <kbd>Adicionar remoto de GitHub</kbd>. Se necessário, siga quaisquer etapas de autenticação e então retorne para o VS Code e repita esta ação.
+
+4. Você deve ver uma lista de todos os seus repositórios no GitHub. Escolha aquele que você criou para este projeto. Se você não vê o seu projeto, cole sua URL do GitHub diretamente. Você pode também ser questionado a dar a esse repositório um nome local. Selecione qualquer nome que desejar.
+
+5. No topo do painel do menu, haverá um lugar para inserir uma **mensagem de commit** (descrição das suas mudanças em arquivos). Digite `commit inicial` e pressione o botão <kbd>Confirmação</kbd> para fazer commit dessas mudanças.
+
+7. Você pode ver uma mensagem te dizendo que você não tem commits "preparados", e te pedindo se você gostaria de prepará-las. Clique <kbd>Sempre</kbd> e continue.
```
Comment (lumamontes) [inline src/content/docs/pt-br/tutorial/1-setup/4.mdx:54]:
```suggestion
7. Você pode ver uma mensagem te dizendo que você não tem commits "preparados", e perguntando se você gostaria de prepará-los. Clique <kbd>Sempre</kbd> e continue.
```

File: src/content/docs/pt-br/tutorial/1-setup/4.mdx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+:::note
+Se você já é familiar com git e tem seu próprio fluxo de trabalho, então crie um novo repositório no GitHub para o seu projeto utilizando seu método preferido. Pule para a próxima página: ["Faça deploy do seu site para a web"](/pt-br/tutorial/1-setup/5/).
+:::
+
+## Crie um repositório no GitHub
+
+Apesar de terem algumas formas de conseguir armazenar o seu código local no GitHub, este tutorial irá te guiar através de um método que não precisa utilizar git na linha de comando. 
+
+1. Entre no GitHub.com em um navegador e clique no <kbd>+</kbd> no lado superior direito da tela para fazer um novo repositório.
```
Comment (lumamontes) [inline src/content/docs/pt-br/tutorial/1-setup/4.mdx:30]:
```suggestion
1. Entre no GitHub.com em um navegador e clique no <kbd>+</kbd> no lado superior direito da tela para criar um novo repositório. 
```

File: src/content/docs/pt-br/tutorial/1-setup/5.mdx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+:::tip[nós utilizaremos...]
+Este tutorial irá utilizar **Netlify**, porém você está livre para utilizar seu serviço de hospedagem preferido para fazer deploy do seu site para a internet.
+:::
+
+## Crie um novo site Netlify
+
+1. Crie uma conta gratuita em [Netlify](https://netlify.com) se você já não a tiver.
+
+    Anote o seu nome de usuário. Você irá ver seu painel de c ontrole e quaisquer sites que criou em `https://app.netlify.com/teams/nome`
```
Comment (lumamontes) [inline src/content/docs/pt-br/tutorial/1-setup/5.mdx:33]:
```suggestion
    Anote o seu nome de usuário. Você irá ver seu painel de controle e quaisquer sites que criou em `https://app.netlify.com/teams/nome`
```

**Comment by Yan-Thomas** [comment]
Thanks for the careful review @lumamontes!

**Pull request merged**